lib/AST/DiagnosticEngine.cpp

```diff
@@ -78,34 +78,22 @@ enum LocalDiagID : uint32_t {
 
 // TODO: categorization
 static const constexpr StoredDiagnosticInfo storedDiagnosticInfos[] = {
-#define ERROR(ID, Options, Text, Signature)                                    \
-  StoredDiagnosticInfo(DiagnosticKind::Error, DiagnosticOptions::Options),
-#define WARNING(ID, Options, Text, Signature)                                  \
-  StoredDiagnosticInfo(DiagnosticKind::Warning, DiagnosticOptions::Options),
-#define NOTE(ID, Options, Text, Signature)                                     \
-  StoredDiagnosticInfo(DiagnosticKind::Note, DiagnosticOptions::Options),
-#define REMARK(ID, Options, Text, Signature)                                   \
-  StoredDiagnosticInfo(DiagnosticKind::Remark, DiagnosticOptions::Options),
+#define DIAG(KIND, ID, Options, Text, Signature)                               \
```

Comment:
To shorten this definition we would need to capitalize all of the `enum class DiagnosticKind` values like the following:
```
enum class DiagnosticKind : uint8_t {
  ERROR,
  WARNING,
  REMARK,
  NOTE
};
```
@xedin WDYT?

Reply:
Which means, that this would require capitalization of these in other places as well right?

Reply:
Naming style of the cases in enums has to stay consistent across repository. I think it's okay to just leave this as is.

Reply:
@xedin makes sense, I'll revert the change that I made for this then, and this will be the only case, where it will be left as is.